### ISSUE TITLE
Tabbing to listbox does not announce listbox title, only item title in Edge and IE11

### ISSUE THREAD

**Issue by jh2he** [FreedomScientific/VFO-standards-support#217]
### Summary

When tabbing into a listbox with JAWS enabled in IE11 and Edge, the label of the listbox is not announced, only the item. The label is announced if there are no items in the lisbox.
This bug does not appear in Chrome, only for IE11 and Edge. JAWS does properly announce the listbox label.

1. Enable JAWS and go to https://www.w3.org/TR/2017/WD-wai-aria-practices-1.1-20170628/examples/listbox/listbox.html
2. Tab to the listbox labelled "Important Features"

### Expected result

JAWS announce listbox label "Important Features" then the default selected item "Proximity of public K-12 schools"

### Actual result

JAWS only announces "Proximity of public K-12 schools"

### Example

https://www.w3.org/TR/2017/WD-wai-aria-practices-1.1-20170628/examples/listbox/listbox.html

#### JAWS version and build number
2019.1906.10 ILM

 
#### Operating System and version
Windows 10

#### Browser and version:
IE 11.239.18362.0
Edge 44.18362.1.0

Bug does not repro in Chrome.

**Comment by lamador** [comment]
The issue was not reproducible using the latest version of JAWS (2020.2006.12), Edge (83.0.478.45) and IE11 (11.836.18362.0) on Windows 10.
